src/components/molecules/Visualizer/Widget/Storytelling/index.tsx

```diff
@@ -201,19 +195,16 @@ const Menu = styled.div<{ menuOpen?: boolean }>`
   border-radius: ${metricsSizes["s"]}px;
   display: ${({ menuOpen }) => (!menuOpen ? "none" : "")};
```

Comment:
I think `unset` value is more appropriate than an empty string, like this:
`display: ${({ menuOpen }) => (!menuOpen ? "none" : "unset")};`